compiler/src/dotty/tools/backend/jvm/BCodeHelpers.scala

```diff
@@ -340,11 +527,16 @@ trait BCodeHelpers extends BCodeIdiomatic with BytecodeWriters {
       )
 
       emitAnnotations(mirrorMethod, others)
-      emitParamAnnotations(mirrorMethod, m.info.params.map(_.annotations))
+      val params: List[Symbol] = Nil // backend uses this to emit annotations on parameter lists of forwarders
+      // to static methods of companion class
+      // in Dotty this link does not exists: there is no way to get from method type
+      // to inner symbols of DefDef
```

Comment:
This is not true anymore now that we have paramSymss, can you update the comment to mention that?

Reply:
Updated comment